pkg/api/response/response.go

```diff
@@ -187,6 +188,26 @@ func JSONStreaming(status int, body interface{}) StreamingResponse {
 	}
 }
 
+// JSONDownload creates a JSON response indicating that it should be downloaded.
+func JSONDownload(status int, body interface{}, filename string) *NormalResponse {
+	return JSON(status, body).SetHeader("Content-Disposition", fmt.Sprintf(`attachment;filename="%s"`, filename))
+}
+
+// YAML creates a YAML response.
+func YAML(status int, body interface{}) *NormalResponse {
+	b, err := yaml.Marshal(body)
+	if err != nil {
+		return Error(http.StatusInternalServerError, "body yaml marshal", err)
+	}
+	// As of now, application/yaml is downloaded by default in chrome regardless of Content-Disposition, so we use text/yaml instead.
+	return Respond(status, b).SetHeader("Content-Type", "text/yaml")
+}
+
+// YAMLDownload creates a YAML response indicating that it should be downloaded.
+func YAMLDownload(status int, body interface{}, filename string) *NormalResponse {
+	return YAML(status, body).SetHeader("Content-Type", "application/yaml").SetHeader("Content-Disposition", fmt.Sprintf(`attachment;filename="%s"`, filename))
```

Comment:
The same here too

```suggestion
	return YAML(status, body).
		SetHeader("Content-Type", "application/yaml").
		SetHeader("Content-Disposition", fmt.Sprintf(`attachment;filename="%s"`, filename))
```

Reply:
Fixed b68338c787c7dc807a33b6635c6ad130e7dcf3dd